test/vm/scripts/kernel_download.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [[ ! $# -ge 2 ]]; then
+  echo "Usage: bash kernel_download.sh BASE_OUTPUT_DIR KERNELS_FILE FORCE_DOWNLOAD"
+  exit 1
+fi
+
+BASE_OUTPUT_DIR="${1}";
+KERNELS_FILE="${2}";
+FORCE_DOWNLOAD="${3}";
+
+set -eou pipefail
+
+HEADERS_DIR="${BASE_OUTPUT_DIR}/headers/";
+KERNELS_DIR="${BASE_OUTPUT_DIR}/kernels/";
+
+if [[ ! -z ${FORCE_DOWNLOAD} || ! -d "${HEADERS_DIR}" || ! -d "${KERNELS_DIR}" || -z "$(ls -A ${HEADERS_DIR})" || -z "$(ls -A ${KERNELS_DIR})" ]] 
+then
+  rm -rf ${HEADERS_DIR}; mkdir -p "${HEADERS_DIR}";
+  rm -rf ${KERNELS_DIR}; mkdir -p "${KERNELS_DIR}";
+
+  cat "${KERNELS_FILE}" | jq -r '.headers[]' | sed 's/\"//g' > /tmp/headers
```

Comment:
Is the sed part removing quotes? Is this needed? I ran the commands locally and the `-r` parameter on jq should take care of it.